### PR TITLE
Updated template to contail all the details our deliverables version …

### PR DESCRIPTION
**Pull request by Roger-Chavez**
…of our stories need, using Chris V's template

### Key features
* New Template, more details to satisfy requirements for deliverables 

### Future Improvements
* none  

### Notes
none

### CODE REVIEW

**Review by ChrisRzech — CHANGES_REQUESTED** [review]
Why was the pull request template deleted?

**Comment by Roger-Chavez** [comment]
> Why was the pull request template deleted?

It was deleted from the ISSUES_TEMPLATE folder because that was a copy. That's where both (pr and issue) templates were created when I made them on github, but I've since moved a copy of the pull request template out, the template you see above is the one that's still there and the other was the original which was supposed to be trashed from the get go, but wasn't.

**Comment by ChrisRzech** [comment]
Oh, forgot david pointed this out to me a few weeks ago. 👍

**Review by ChrisRzech — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/user-stories-template.md [hunk 1]
```diff
@@ -1,22 +1,12 @@
----
-name: User Stories template
-about: The purpose of this template is to help make a description of a software feature
-  from an end-user perspective.
-title: ''
-labels: ''
-assignees: ''
+<!-- ### == <h3>, * == <ul> ,  1. == <ol> -->
```
Comment (ChrisRzech) [inline .github/ISSUE_TEMPLATE/user-stories-template.md:1]: Why is there a comment here?

Comment (Roger-Chavez) [reply .github/ISSUE_TEMPLATE/user-stories-template.md:1]: Shows the equivalent syntax from html to markdown so they don't have to be looked up

Comment (ChrisRzech) [reply .github/ISSUE_TEMPLATE/user-stories-template.md:1]: Doesn't show up on the PR?

Comment (ChrisRzech) [reply .github/ISSUE_TEMPLATE/user-stories-template.md:1]: As in, if I make a PR, will I have to remove that comment?

Comment (Roger-Chavez) [reply .github/ISSUE_TEMPLATE/user-stories-template.md:1]: No you won't have to remove it, you can leave it, it won't show up when you submit. Only show's when you edit.

Comment (ChrisRzech) [reply .github/ISSUE_TEMPLATE/user-stories-template.md:1]: Perfect 👍 LAND HO!

**Pull request merged**